### PR TITLE
Mkerns/caseflow 868 last updated poa

### PR DESCRIPTION
**Pull request by mkerns355**
Resolves [CASEFLOW-868](https://vajira.max.gov/browse/CASEFLOW-868)

### Description
Added PoaRefresh component to show user the most recent last synced at date for POA data.

### Acceptance Criteria
- [x] Code compiles correctly
- [x] Given a user advances to the Case Details page
  - [x] When they view the POA section 
  - [x] Then the last updated date displays 
- [x] Work for this should be placed behind the poa_sync_date Feature Toggle

### Testing Plan
1. Enable the poa sync date feature toggle `FeatureToggle.enable!(:poa_sync_date) `
2. Go to Case Details page
2. Scroll to Power Of Attorney Details Section
3. Verify that the last sync date and text is visable

### User Facing Changes
 - [x] Screenshots of UI changes added to PR & Original Issue

 BEFORE|AFTER
 
image
|
image

### Storybook Story
*For Frontend (Presentational) Components*
* [x] Add a [Storybook](https://github.com/department-of-veterans-affairs/caseflow/wiki/Documenting-React-Components-with-Storybook) file alongside the component file (e.g. create `MyComponent.stories.js` alongside `MyComponent.jsx`)
* [x] Give it a title that reflects the component's location within the overall Caseflow hierarchy
* [x] Write a separate story (within the same file) for each discrete variation of the component

### CODE REVIEW

**Comment by bgantick** [comment]
@mkerns355 Reminder to update the description, make sure this only includes frontend for displaying the last synced at date, and ensure everything is behind a feature toggle

File: app/views/queue/index.html.erb [hunk 1]
```diff
@@ -31,6 +31,7 @@
       editNodDate: FeatureToggle.enabled?(:edit_nod_date, user: current_user),
       fnod_badge: FeatureToggle.enabled?(:fnod_badge, user: current_user),
       fnod_banner: FeatureToggle.enabled?(:fnod_banner, user: current_user),
+      view_poa_refresh: FeatureToggle.enabled?(:view_poa_refresh, user: current_user),
```
Comment (bgantick) [inline app/views/queue/index.html.erb:34]: Per the AC in the Jira story, this should be `poa_sync_date`

File: client/app/queue/components/PoaRefresh.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+export const textStyling = css({
+  display: 'flex',
+  justifyContent: 'space-between'
+});
+
+export const PoaRefresh = ({ powerOfAttorney }) => {
+  const poaSyncInfo = {
+    poaSyncDate: formatDateStr(powerOfAttorney.poa_last_synced_at)
+  };
```
Comment (ajspotts) [inline client/app/queue/components/PoaRefresh.jsx:19]: 🎉

**Review by ajspotts — APPROVED** [review]
Feature toggle ✅ 
Functionality ✅ 
LGTM!

**Pull request merged**